lib/board.rb

```diff
@@ -0,0 +1,55 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+  attr_reader   :player
+  # def cells
+  #   []
+  # end
+
+  def initialize
+    @cells = Array.new(9," ")
+  end
+
```

Comment:
I would just call `self.reset!` here (in `#initialize`) and delegate to that method to create the default empty board.